Source/WebKit/Shared/cf/CoreIPCSecTrust.mm

```diff
@@ -566,8 +566,14 @@ static String optionalArrayOfDataHelper(std::optional<Vector<CoreIPCData>>& toSe
             },
             [&] (const CoreIPCSecTrustData::PolicyArrayOfData& a) {
                 RetainPtr array = adoptNS([[NSMutableArray alloc] initWithCapacity:a.size()]);
-                for (const auto& d : a)
-                    [array addObject:d.toID().get()];
+                for (const auto& d : a) {
+                    if (RetainPtr nsD = d.toID())
+                        [array addObject:d.toID().get()];
+                    else {
+                        RELEASE_LOG_ERROR(IPC, "CoreIPCSecTrustData had an null value in policy dictionary");
+                        ASSERT_NOT_REACHED();
```

Comment:
I think this change is fine as an initial step to address the crash, but I'd like to see some kind of validation on the "send" side to see why we are populating IPC data with null values. Identifying which category is the culprit is a good first step.

Reply:
There is no case on the send side where we (WebKit) are populating IPC data with these null values. This is related to an external program that you can find more information about in the radar.